crates/sui/tests/cli_tests.rs

```diff
@@ -3021,9 +3113,11 @@ async fn test_get_owned_objects_owned_by_address_and_check_pagination() -> Resul
 
 #[tokio::test]
 async fn test_linter_suppression_stats() -> Result<(), anyhow::Error> {
+    let mut test_cluster = TestClusterBuilder::new().build().await;
+    let context = &mut test_cluster.wallet;
```

Comment:
Updates this test to use a local network to resolve a `chain_id` (required to resolve addresses from `Move.lock`)